components/sup/src/manager/service/mod.rs

```diff
@@ -1003,20 +1003,26 @@ impl fmt::Display for Service {
     }
 }
 
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub enum ConfigRendering {
```

Comment:
Adding some documentation describing what each of these does would be great, and is one of the benefits of using Enums-as-Booleans; it gives you a really nice place to hang documentation off of.

Reply:
Also, since the intention of this enum is to act like a "named boolean", we could make it behave even more like a boolean by deriving `Copy`, which means you wouldn't actually need to derive `Clone`, and wouldn't need to call `.clone()` everywhere you pass this around, which makes for an easier-to-use API.

Clearly calling that out in the documentation string is also a good idea, because implementing `Copy` is a Big Deal (TM). We don't want to do any kind of refactoring of this type in the future that adds some kind of extra large-memory member, because then we'd immediately be copying it everywhere. That's _highly_ unlikely in this usecase, but making that very clear can't hurt.

Reply:
Ah, good call.  The `.clone()` thing in 2 places was bugging me, but I wasn't entirely sure how to get around it.